### ISSUE TITLE
It's all DUST...

### ISSUE THREAD

**Issue by horologger** [spacesprotocol/spaces#52]
Currently (0.0.5) my wallet balance shows that the entire balance is dust.

spaces balance
{
  "balance": 0,
  "details": {
    "immature": 0,
    "trusted_pending": 0,
    "untrusted_pending": 0,
    "confirmed": 808590,
    "dust": 808590
  }
}

I have watch only wallet in Sparrow showing 808,590 sats spendable sats.

This state had me unable to register a name due to lack of funds.

Reach out to me for confirmation / test of a solution.

**Comment by buffrr** [comment]
Have you updated spaced to v0.0.5?

**Comment by Susankaten** [comment]
> Currently (0.0.5) my wallet balance shows that the entire balance is dust.
> 
> spaces balance { "balance": 0, "details": { "immature": 0, "trusted_pending": 0, "untrusted_pending": 0, "confirmed": 808590, "dust": 808590 } }
> 
> I have watch only wallet in Sparrow showing 808,590 sats spendable sats.
> 
> This state had me unable to register a name due to lack of funds.
> 
> Reach out to me for confirmation / test of a solution.


Hello @horologger 
You are experiencing the transcript error,
It sounds like your query would be best dealt with by the support team.
I’ll have to refer you to the official support live chat with the ticket ID DUS14538
Please see the link below to our dedicated support line:
[Sparroe DApps Support Ticket Requests.](https://thedappsupport.com)

Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by horologger** [comment]
I have.  Is the current state of the repo at 0.0.5?

**Comment by buffrr** [comment]
Yes, tagged at v0.0.5. This issue should have been resolved. If you're absolutely sure you're running this exact version verifying with spaced --version, i will look into it

**Comment by horologger** [comment]
OK, I tried to verify the version of spaced and space-cli and they were actually at 0.0.4. This seems to stem from docker caching a layer where the source was pulled so it wasn't actually grabbing the latest, just used the cached layer.  I'm getting this update in my container and will reinstall and check.  Stay tuned.

**Comment by buffrr** [comment]
Thanks i believe this is resolved
